chromium_src/components/feature_engagement/public/feature_constants.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/components/feature_engagement/public/feature_constants.cc"
+
+#include "base/feature_override.h"
+
+namespace feature_engagement {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kIPHGMCCastStartStopFeature, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kIPHPasswordsManagementBubbleAfterSaveFeature,
+     base::FEATURE_DISABLED_BY_DEFAULT},
+}});
```

Comment:
Do we not need the android guard here, same as above?

Reply:
Sure do! 🤦‍♂️